transaction/std/src/memo_builder/rth_memo_builder.rs

```diff
@@ -159,7 +159,7 @@ impl MemoBuilder for RTHMemoBuilder {
             .checked_add(1)
             .ok_or(NewMemoError::LimitsExceeded("num_recipients"))?;
         self.last_recipient = ShortAddressHash::from(recipient);
-        Ok(if let Some(cred) = &self.sender_cred {
+        Ok(Some(if let Some(cred) = &self.sender_cred {
```

Comment:
Nitpick (readability): It might be slightly less intimidating to pull the value going into the `Ok()` out and trust the compiler to do the right thing. I realize that was in the original PR, and it's not strictly necessary, but two layers of `if let Some(foo)` inside an `Ok()` is a bit much...

Reply:
yeah okay i can do that